astropy/modeling/core.py

```diff
@@ -1352,6 +1352,9 @@ def _format_str(self, keywords=[]):
         else:
             columns = [getattr(self, name).value
                        for name in self.param_names]
+        # check for models without parameters
+        if not columns:
+            return '\n'.join(parts)
```

Comment:
My preference here would be to change this to
```python
if columns:
    param_table = Table(columns, names=self.param_names)
    parts.append(indent(str(param_table), width=4))
```
No repetitive return statements that way.